_topic_map.yml

```diff
@@ -203,16 +203,13 @@ Name: Nodes
 Dir: nodes
 Distros: openshift-rosa,openshift-dedicated
 Topics:
-- Name: About machine pools and autoscaling
-  File: nodes-machinepools-autoscaling-about
+- Name: About machine pools
+  File: nodes-machinepools-about
 - Name: Managing worker nodes
   File: rosa-managing-worker-nodes
   Distros: openshift-rosa
-- Name: Enabling autoscaling nodes on a cluster
-  File: nodes-enabling-autoscaling-nodes
-- Name: Disabling autoscaling nodes on a cluster
-  File: nodes-disabling-autoscaling-nodes
-
+- Name: About autoscaling nodes on a cluster
```

Comment:
I see that this is an existing term. But, _autoscaling nodes_ threw me off for a moment. I assumed that _autoscaling_ was an adjective; that _autoscaling nodes_ was a type of node. Probably outside the context of this PR.